tests/capture/test_capture_qnode.py

```diff
@@ -65,8 +65,9 @@ def circuit():
         circuit()
 
     jax.make_jaxpr(partial(circuit, shots=50))()  # should run fine
-    res = circuit(shots=50)
-    assert qml.math.allclose(res, jax.numpy.zeros((50,)))
+    with pytest.raises(NotImplementedError, match="override shots are not yet supported"):
```

Comment:
These will need to be updated if the error messages are updated per my earlier suggestion.